diff --git a/node_modules/@tiptap/extension-invisible-characters/dist/index.cjs b/node_modules/@tiptap/extension-invisible-characters/dist/index.cjs
--- a/node_modules/@tiptap/extension-invisible-characters/dist/index.cjs
+++ b/node_modules/@tiptap/extension-invisible-characters/dist/index.cjs
@@ -189,7 +189,6 @@ var createDecorationWidget = (pos, type, content) => {
   };
   return import_view2.Decoration.widget(pos, createElement, {
     key: type,
-    marks: [],
     side: 1e3
   });
 };
diff --git a/node_modules/@tiptap/extension-invisible-characters/dist/index.js b/node_modules/@tiptap/extension-invisible-characters/dist/index.js
--- a/node_modules/@tiptap/extension-invisible-characters/dist/index.js
+++ b/node_modules/@tiptap/extension-invisible-characters/dist/index.js
@@ -157,7 +157,6 @@ var createDecorationWidget = (pos, type, content) => {
   };
   return Decoration.widget(pos, createElement, {
     key: type,
-    marks: [],
     side: 1e3
   });
 };
diff --git a/node_modules/@tiptap/extension-invisible-characters/src/plugin/utils/create-decoration-widget.ts b/node_modules/@tiptap/extension-invisible-characters/src/plugin/utils/create-decoration-widget.ts
--- a/node_modules/@tiptap/extension-invisible-characters/src/plugin/utils/create-decoration-widget.ts
+++ b/node_modules/@tiptap/extension-invisible-characters/src/plugin/utils/create-decoration-widget.ts
@@ -16,7 +16,6 @@ export const createDecorationWidget = (pos: number, type: string, content?: stri
 
   return Decoration.widget(pos, createElement, {
     key: type,
-    marks: [],
     side: 1000,
   })
 }
